### PR TITLE
fix(ipUpgrader): fix bug where upgrader is unable to use new cpio files

### PR DESCRIPTION
**Pull request by pkhansen**
This is meant as a low risk effort to make the upgrader work with the new cpio files used for the ip cameras. In the future we should address how we read and process cpio files and make sure to support the new image format. For now the following has been done.

* Added additional timeout

* Using cpio-reader callback to progress also when going through unused files

### CODE REVIEW

File: src/components/upgrader/ipUpgrader.ts [hunk 1]
```diff
@@ -438,7 +438,7 @@ export default class IpCameraUpgrader extends EventEmitter implements IDeviceUpg
         Logger.warn(errMsg, this.className);
         this.emit(CameraEvents.UPGRADE_FAILED, errMsg);
         reject(errMsg);
-      }, 10000);
+      }, 42000);
```
Comment (eskilgh) [inline src/components/upgrader/ipUpgrader.ts:441]: We really need this long to read the cpio?

File: src/components/upgrader/ipUpgrader.ts [hunk 2]
```diff
@@ -438,7 +438,7 @@ export default class IpCameraUpgrader extends EventEmitter implements IDeviceUpg
         Logger.warn(errMsg, this.className);
```
Comment (eskilgh) [inline src/components/upgrader/ipUpgrader.ts:438]: While you are here, can you fix the errMsg to not hard code the seconds timeout (line 437)?

**Review by eskilgh — APPROVED** [review]
👏

File: src/components/upgrader/ipUpgrader.ts (hunk 1 above)
Comment (pkhansen) [reply src/components/upgrader/ipUpgrader.ts:441]: Don't think so. Was told that this was used elsewhere, but honestly shouldn't take that long.

File: src/components/upgrader/ipUpgrader.ts (hunk 2 above)
Comment (pkhansen) [reply src/components/upgrader/ipUpgrader.ts:440]: Yup, good thinking :D

**Pull request merged**